packages/bubble/Bubble.tsx

```diff
@@ -0,0 +1,90 @@
+import { h, Component, prop } from 'skatejs';
+import styles from './Bubble.scss';
+import { css } from '../../utils/css';
+
+const BubbleTypes = {
+  top: 'top',
+  left: 'left',
+  right: 'right',
+  bottom: 'bottom',
+};
+
+type BubbleTypesType = typeof BubbleTypes;
+
+// public
+interface BubbleProps extends JSX.HTMLProps<HTMLElement | any> {
+  type?: keyof BubbleTypesType,
+  isDisplayed?: boolean,
+  disableAutoShowHide?: boolean
+}
+
+export class Bubble extends Component<BubbleProps> {
+  _props: BubbleProps;
+  static get is() { return 'bl-bubble' }
+  static get props() {
+    return {
+      type: prop.string({
+        attribute: true
```

Comment:
Do we need reflect ```type``` into attribute? We dont change it in lifecycle, or yes? The same for ```isDisplayed``` and ```disableAutoShowHide```